### PR TITLE
bug fix: fixes variable visibility for interoperability

### PR DESCRIPTION
**Pull request by youvegotmail-1998**
makes counter variables public for interoperability

### CODE REVIEW

**Comment by youvegotmail-1998** [comment]
@lbelyaev please see PR for consideration

File: contracts/XENFT.sol [hunk 1]
```diff
@@ -32,9 +32,10 @@ contract XENFT is IXENTorrent, IXENProxying, ERC721("XENFT", "XENFT") {
     // original contract marking to distinguish from proxy copies
     address private immutable _original;
     // ever increasing counter for NFT tokenIds, also used as salt for proxies' spinning
-    // TODO: make the next 2 public ???
-    uint256 private _tokenIdCounter = LIMITED_SERIES_COUNT;
```
Comment (nibty) [inline contracts/XENFT.sol:36]: I believe these vars should be renamed with the leading underscore removed since they would no longer be private.

**Review by nibty — COMMENTED** [review]
Please rebase from master to include a fix to the code linting actions.

**Comment by youvegotmail-1998** [comment]
@nibty all set

**Comment by lbelyaev** [comment]
@youvegotmail-1998 thanks for your contribution!

**Pull request merged**